cmd/kuberhealthy/main.go

```diff
@@ -34,13 +34,14 @@ import (
 )
 
 // status represents the current Kuberhealthy OK:Error state
-var kubeConfigFile = filepath.Join(os.Getenv("HOME"), ".kube", "config")
+var kubeConfigFile = filepath.Join(os.Getenv("KUBECONFIG"))
```

Comment:
Can we leave this line as:
`var kubeConfigFile = filepath.Join(os.Getenv("HOME"), ".kube", "config")`

and then add an override if the `KUBECONFIG` environment variable is set?   This way users who do not have a `KUBECONFIG` variable set will not be broken and users who do use it will see it take effect.